assets/css/global-custom.css

```diff
@@ -500,6 +504,10 @@ body{
     padding: .5rem .75rem;
 }
 
+.card-height{
+    height: 400px;
+}
+
```

Comment:
Pick a better class name, `card-height` doesn't describe what this class actually does. For an example the above class you have created `rounded-24px` name it self is pretty self explanatory. Change the class name into something like that. :)

Reply:
I changed it. Could you have a look at it?

Reply:
Resolved with commit 86907d68ec4cc3f4f2d824ca8c5db80cc1136a87. please review